diagnostic.md

```diff
@@ -21,13 +21,18 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com
 ```
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function (event) {
+  return $.ajax({
+  url:'http://www.example.com/donuts',
+  method: 'GET',
+  });
+};
```

Comment:
What is `event` here? A good rule of thumb is, if you are not using the variable that's been passed into your function, you don't need it.